bears/python/BanditBear.py

```diff
@@ -0,0 +1,67 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Setting import typed_list
+
+
+@linter(executable='bandit')
+class BanditBear:
+    """
+    Performs security analysis on Python source code, utilizing the ``ast``
+    module from the Python standard library.
```

Comment:
nobody really cares how it does it IMO, rather elaborate a bit more on what it can do!